4-Alphabet/Alphabet.py

```diff
@@ -0,0 +1,18 @@
+def print_simbols(columns):
+	counter = 0
+	result_2 = ''
+	for number in range(range_start,range_end):
+		number = chr(number)+ ' '
+		if counter % columns == 0:
+			result_2 = result_2 + '\n'
+		result_2 = result_2 + number 
+		counter = counter + 1
+	print(str(result_2))
+#print("Choose ASCII symbols and press enter: 1 - All, 2 - Latin, 3 - Cyrillic. To exit press Ctrl-C.\n")
```

Comment:
Does it work as expected in [issue comment](https://github.com/ArtemDomnikoff/Repository/issues/14#issuecomment-560423463)? 
@ArtemDomnikoff